### PR TITLE
Update README ad CONTRIBUTING files

### PR DESCRIPTION
**Pull request by aneuwald-ctw**
Update README.md and CONTRIBUTING.md files to have a more clear information about the project.

Also normalize the name Foxbox, some parts of the project was using FoxBox.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (laisspportugal) [inline CONTRIBUTING.md:1]: Why removing dependencies?

**Comment by laisspportugal** [comment]
Can you keep the pattern Foxbox?

File: CONTRIBUTING.md
Comment (aneuwald-ctw) [reply CONTRIBUTING.md:1]: It was moved to README.md

**Review by laisspportugal — APPROVED** [review]
Looks good to me

**Pull request merged**